waspc/README.md

```diff
@@ -121,20 +121,19 @@ NOTE: Reload page if blank.
 3. Do a change in the codebase (most often in `src/` or `cli/src/` or `data/`) (together with tests if that makes sense: see "Tests").
    Fix any errors shown by HLS/`ghcid`.
    Rinse and repeat.
-4. If you did a bug fix, added new feature or did a breaking change, add short info about it to Changelog.md. Also, bump version in waspc.cabal and ChangeLog.md if needed -> check the version of latest release when doing it. If you are not sure how to decide which version to go with, check later in this file instructions on it.
-5. Once close to done, run `cabal test` to confirm that the project's tests are passing (both new and old).
-6. If needed, confirm that `examples/todoApp/` is working correctly by running `cabal build` first, to build the wasp executable, and then by running that executable with `cabal run wasp-cli start` from the `examples/todoApp/` dir -> this will run the web app in development mode with the current version of your Wasp code.
-   Manually inspect that app behaves ok: In the future we will add automatic integration tests, but for now testing is manual.
-7. Squash all the commits into a single commit (or a few in case it makes more sense) and create a PR.
+4. Run `cabal test` to confirm that the project's unit and integration tests are passing (both new and old).
```

Comment:
Should we rather refer to the `run` script? I like that script.

I'd prefer if we listed `run` commands for:
- Running unit tests.
- Running e2e tests.
- Running headless tests.

Reply:
Hmmm maybe. I also love `run` script, but in the docs I always treated it as an "optional", "extra" thing.

I am not sure how exactly I would rewrite this to use `run` script I have to admit. I am a bit out of context, and I know I was happy with this. So I would suggest leaving it as it is, as it is also relatively simple, and we can see about optimizing stuff with `run` script instructions in the future.